src/library_syscall.js

```diff
@@ -227,7 +227,7 @@ var SyscallsLibrary = {
   },
   __syscall5: function(which, varargs) { // open
     var pathname = SYSCALLS.getStr(), flags = SYSCALLS.get(), mode = SYSCALLS.get() // optional TODO
-    var stream = FS.open(pathname, flags, mode);
+    var stream = FS.open(pathname, flags | {{{ cDefine('O_LARGEFILE') }}}, mode);
```

Comment:
Looks like musl started to default to largefile, so you changed emscripten to as well?

I wonder if that affects performance, does it change all offsets to 64-bit? Seems like in asm.js and wasm32, we don't need largefile?